lib/knapsack_pro/runners/queue/cucumber_runner.rb

```diff
@@ -3,6 +3,8 @@ module Runners
     module Queue
       class CucumberRunner < BaseRunner
         def self.run(args)
+          # ENV['RAILS_ENV'] = ENV['RACK_ENV'] = 'test'
```

Comment:
We are good to remove comments now since we establish solution with `$PROGRAM_NAME`.